src/io/inputs.jl

```diff
@@ -243,10 +243,7 @@ end
 Constructor for YAXArray with all entries equal zero.
 """
 function ZeroDataCube(T=Float64; kwargs...)::YAXArray
-    return DataCube(
-        zeros(T, Tuple(length(kwargs[name]) for name in Symbol.(keys(kwargs))));
-        kwargs...
-    )
+    return DataCube(zeros(T, [length(kwargs[name]) for (name, _) in kwargs]...); kwargs...)
```

Comment:
`length(val) for (name, val) in kwargs`

`zeros(T, length.(kwargs)...)` might work too